src/test/java/com/example/module3/AirportTest.java

```diff
@@ -42,66 +39,41 @@
     public void testGetTransportMilitaryPlanes() {
         Airport airport = new Airport(planes);
         List<MilitaryPlane> transportMilitaryPlanes = airport.filterMilitaryPlanesByType(MilitaryType.TRANSPORT);
-        boolean flag = false;
-        for (MilitaryPlane militaryPlane : transportMilitaryPlanes) {
-            if ((militaryPlane.getType() == MilitaryType.TRANSPORT)) {
-                flag = true;
-                break;
-            }
-        }
-        Assert.assertEquals(flag, true);
+        boolean containsTransportOnly = TestServices.transportMilitaryPlane(transportMilitaryPlanes);
+        Assert.assertTrue(containsTransportOnly, "Airport contains not only transport planes");
     }
 
     @Test
     public void testGetPassengerPlaneWithMaxCapacity() {
         Airport airport = new Airport(planes);
         PassengerPlane expectedPlaneWithMaxPassengersCapacity = airport.getPassengerPlaneWithMaxPassengersCapacity();
-        Assert.assertTrue(expectedPlaneWithMaxPassengersCapacity.equals(planeWithMaxPassengerCapacity));
+        Assert.assertEquals(planeWithMaxPassengerCapacity, expectedPlaneWithMaxPassengersCapacity);
     }
 
     @Test
     public void testSortedPlaneByMaxLoadCapacity() {
         Airport airport = new Airport(planes);
-        airport.sortPlanesByMaxLoadCapacity();
+        airport.sortPlanesBy(PlaneComparator.LOAD_CAPACITY);
         List<? extends Plane> planesSortedByMaxLoadCapacity = airport.getPlanes();
-
-        boolean nextPlaneMaxLoadCapacityIsHigherThanCurrent = true;
-        for (int i = 0; i < planesSortedByMaxLoadCapacity.size() - 1; i++) {
-            Plane currentPlane = planesSortedByMaxLoadCapacity.get(i);
-            Plane nextPlane = planesSortedByMaxLoadCapacity.get(i + 1);
-            if (currentPlane.getMaxLoadCapacity() > nextPlane.getMaxLoadCapacity()) {
-                nextPlaneMaxLoadCapacityIsHigherThanCurrent = false;
-                break;
-            }
-        }
+        boolean nextPlaneMaxLoadCapacityIsHigherThanCurrent = TestServices
+                .sortedPlaneByMaxLoadCapacity(planesSortedByMaxLoadCapacity);
         Assert.assertTrue(nextPlaneMaxLoadCapacityIsHigherThanCurrent);
     }
 
     @Test
     public void testHasAtLeastOneBomberInMilitaryPlanes() {
         Airport airport = new Airport(planes);
         List<MilitaryPlane> bomberMilitaryPlanes = airport.filterMilitaryPlanesByType(MilitaryType.BOMBER);
-        boolean flag = false;
-        for (MilitaryPlane militaryPlane : bomberMilitaryPlanes) {
-            if ((militaryPlane.getType() == MilitaryType.BOMBER)) {
-                flag = true;
-            } else {
-                Assert.fail("Test failed!");
-            }
-        }
+        boolean containsOnlyBomber =TestServices.hasAtLeastOneBomberInMilitaryPlanes(bomberMilitaryPlanes);
```

Comment:
Method has a name `test has at least one bomber in military planes`, but you return boolean `contains only bomber`.
These are two different things.